src/libdrakvuf/libdrakvuf.h

```diff
@@ -223,6 +223,34 @@ struct drakvuf_trap {
     void *data;
 };
 
+
+////////////////////////////////////////////////////////////////////////////
+
+// IMHO these definitions must be placed within another file, named
+// libdrakvuf-windows.h or something similar
+
+// For get_previous_mode...
+typedef enum _MODE {
```

Comment:
Yea, that's a bridge we will have to cross when I add Linux support.. ;) For now it's fine to leave them in libdrakvuf.h